wireapp/wire-ios
# chore: use datadog fork and compatible with Xcode 13.2.1

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

We use directly the datadog github's framework in Carthage. To be safe we should use a fork.

### Solutions

Use wireapp fork + compatible version with our Xcode version hosted on the fork.

### Testing

#### How to Test

The project should build as usual BUT not compile from source datadog (download xcframework)

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
